python/py_init.cpp

```diff
@@ -39,17 +40,15 @@ PyObject* get_gflag(PyObject* self, PyObject* args) {
         throw PyException("get_gflag args incorrect");
     }
     std::string flag_name = c_name;
-    if (flag_name == "task_mode") {
+    if (flag_name == "visible_radius") {
+        return Py_BuildValue("i", FLAGS_visible_radius);
+    } else if (flag_name == "task_mode") {
         return PyString_FromString(FLAGS_task_mode.c_str());
-    } 
-    #ifdef XWORLD3D
-    else if (flag_name == "x3_task_mode") {
-        return PyString_FromString(FLAGS_x3_task_mode.c_str());
-    } else if (flag_name == "x3_reaching_distance") {
-        return Py_BuildValue("d", FLAGS_x3_reaching_distance);
-    }
-    #endif
-    else if (flag_name == "curriculum") {
+        //    } else if (flag_name == "x3_task_mode") {
```

Comment:
why comment out x3_task_mode and x3_reaching_distance?

Reply:
They are no longer used.